pepys_import/core/store/data_store.py

```diff
@@ -44,6 +45,23 @@
 DEFAULT_DATA_PATH = os.path.join(PEPYS_IMPORT_DIRECTORY, "database", "default_data")
 USER = getuser()  # Login name of the current user
 
+# Constant Lists of revisions - list all files in the directory
```

Comment:
Could you add a more detailed comment here - saying that this is a list of revisions known to this version of Pepys, and by putting it in the top of the file (rather than in a function), it is computed once at file load, thus making the DataStore constructor more efficient